src/core/print_ps.cpp

```diff
@@ -258,7 +258,8 @@ CalcAllValues(bool PrintLandscape, bool PrintDoCont, bool overview, double PageW
 		width *= fieldwidth/fullwidth;
 		height *= fieldheight/fullheight;
 	}
-	return { width, height, real_width, real_height, field_x, field_y, field_w, field_h, stage_field_x, stage_field_y, stage_field_w, stage_field_h, step_size, spr_step_size, step_width };
+	return std::tuple<float, float, float, float, float, float, float, float, float, float, float, float, float, float, short>(
```

Comment:
Holy crap, are you sure there's no way to configure the Visual Studio compiler so we don't have to do this?